airflow/gcp/hooks/mlengine.py

```diff
@@ -275,3 +275,19 @@ def get_model(self, project_id: str, model_name: str) -> Optional[Dict]:
                 self.log.error('Model was not found: %s', e)
                 return None
             raise
+
+    def delete_model(self, project_id: str, model_name: str) -> None:
```

Comment:
Here we need a fallback for project id, but I prefer to do it in a separate PR for whole integration. I prefer to maintain uniformity at the expense of functionality